fred2/bgbitmapdlg.cpp

```diff
@@ -278,9 +278,9 @@ void bg_bitmap_dlg::create()
 		m_skybox_heading = m_skybox_heading + 360;
 
 
-	for(i=0; i<MAX_NEB2_BITMAPS; i++){
-		if(strlen(Neb2_bitmap_filenames[i]) > 0){ //-V805
-			((CComboBox*)GetDlgItem(IDC_NEB2_TEXTURE))->AddString(Neb2_bitmap_filenames[i]);
+	for (i = 0; i < (int)Neb2_bitmap_filenames.size(); i++) {
+		if (Neb2_bitmap_filenames[i].length() > 0) {
```

Comment:
Very minor, but I'm wonder if all these `len > 0` checks should be kept. It was only like that because before, that's what the whole array was initialized with, forcing it to assume "" was an unused slot.  Now, "" as a name could be technically valid, but obviously that's a terrible idea...

Reply:
Hmm. While technically valid I think `stuff_string()` will error on an empty string. So yeah, I think it's impossible to have an item in the list with a name of zero length. These can probably go.

Reply:
Tested and parsing fails on an empty filename string, so I have removed the length checks.